test/UnitTest/Microsoft.AspNet.OData.Test/PublicApi/Microsoft.AspNet.OData.PublicApi.bsl

```diff
@@ -3312,14 +3313,16 @@ public class Microsoft.AspNet.OData.Formatter.Serialization.SelectExpandNode {
 	public SelectExpandNode (SelectExpandNode selectExpandNodeToCopy)
 	public SelectExpandNode (Microsoft.OData.Edm.IEdmStructuredType structuredType, ODataSerializerContext writeContext)
 	public SelectExpandNode (Microsoft.OData.UriParser.SelectExpandClause selectExpandClause, Microsoft.OData.Edm.IEdmStructuredType structuredType, Microsoft.OData.Edm.IEdmModel model)
-	public SelectExpandNode (Microsoft.OData.UriParser.SelectExpandClause selectExpandClause, Microsoft.OData.Edm.IEdmStructuredType structuredType, Microsoft.OData.Edm.IEdmModel model, bool expandedReference)
 
```

Comment:
Isn't removing this overload a breaking change? Why are we convinced it's okay?

Reply:
It wouldn't be as the above constructor is not released yet. Expand on $ref PR introduced that constructor which I refactored.

---
In reply to: [308507235](https://github.com/OData/WebApi/pull/1738#discussion_r308507235) [](ancestors = 308507235)

Reply:
I believe that I had it fixed in the $ref PR. I will know when I rebase. 

---
In reply to: [308843120](https://github.com/OData/WebApi/pull/1738#discussion_r308843120) [](ancestors = 308843120,308507235)

Reply:
Please verify after rebase.

---
In reply to: [326414044](https://github.com/OData/WebApi/pull/1738#discussion_r326414044) [](ancestors = 326414044,308843120,308507235)